src/app_task.cpp

```diff
@@ -155,6 +165,17 @@ CHIP_ERROR AppTask::Init()
 	k_timer_init(&sFunctionTimer, &AppTask::FunctionTimerTimeoutCallback, nullptr);
 	k_timer_user_data_set(&sFunctionTimer, this);
 
+	k_timer_init(
+		&sSensorTimer,
+		[](k_timer *timer) {
+			AppEvent event;
+			event.Type = AppEventType::FetchMeas;
+			event.TimerEvent.Context = k_timer_user_data_get(timer);
```

Comment:
Isn't this member supposed to be modified for Timer event only? And is any user data set to the timer at all?